app/components/elements/forms/select_field_component.rb

```diff
@@ -3,9 +3,8 @@
 module Elements
   module Forms
     # Component for form select fields
-    # rubocop:disable Metrics/ParameterLists
```

Comment:
If the disable applies to all of a method or a class, place it on the `def` or `class` line and it will scope appropriately *and* not require an explicit enable below.